src/Security/src/Authorization.Certificate/CertificateHttpClientBuilderExtensions.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Steeltoe.Common;
+using Steeltoe.Common.Certificates;
+using Steeltoe.Common.Configuration;
+
+namespace Steeltoe.Security.Authorization.Certificate;
+
+public static class CertificateHttpClientBuilderExtensions
+{
+    /// <summary>
+    /// Configures <see cref="CertificateOptions" /> representing the application instance and attaches the certificate to outbound requests.
+    /// </summary>
+    /// <param name="httpClientBuilder">
+    /// The <see cref="IHttpClientBuilder" /> to add a client certificate to.
+    /// </param>
+    public static IHttpClientBuilder AddClientCertificateForAppInstance(this IHttpClientBuilder httpClientBuilder)
+    {
+        return AddClientCertificate(httpClientBuilder, CertificateConfigurationExtensions.AppInstanceIdentityCertificateName);
+    }
+
+    /// <summary>
+    /// Configures named <see cref="CertificateOptions" /> and attaches the certificate to outbound requests.
+    /// </summary>
+    /// <param name="httpClientBuilder">
+    /// The <see cref="IHttpClientBuilder" /> to add a client certificate to.
```

Comment:
```suggestion
    /// The <see cref="IHttpClientBuilder" /> to configure.
```
We're not adding a certificate to the builder, but to the outbound requests it acts on.